custom-eureka-client/src/main/resources/application.properties

```diff
@@ -0,0 +1,8 @@
+spring.application.name=test

```

Comment:
remove special character at the end of each line

Reply:
updated.